db/migrate/20220726073301_create_recipes.rb

```diff
@@ -0,0 +1,13 @@
+class CreateRecipes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipes do |t|
+      t.string  :name
+      t.integer :preparation_time, default: 0
+      t.integer :cooking_time, default: 0
```

Comment:
The data type for `preparation_time` and `cooking_time` should be time and not `integer`.